satellites/main.yaml

```
@@ -0,0 +1,8 @@
+
+sources:
+
+  GOES16:
```

Just to be consistent with the other files, I'd skip the empty lines.

```suggestion
sources:
  GOES16:
```